### PR TITLE
Remove unused attributes of countries in update-third-party-libs.sh

### PR DESCRIPTION
**Pull request by Baiqiang**
Only name and alpha2 attributes are used. I changed the update script to reduce its size from 63kb to 15kb.

### CODE REVIEW

File: update-third-party-libs.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 (cd both/components/; wget -N http://www.jflei.com/jChester/jChester.js)
 
-(cd both/components/; curl -s https://raw.githubusercontent.com/OpenBookPrices/country-data/master/data/countries.json | sed -e '1s/^/countries = /' -e '$a\;' > countries.js)
+(cd both/components/; curl -s https://raw.githubusercontent.com/OpenBookPrices/country-data/master/data/countries.json | sed -e '1s/^/JSON.stringify(/' -e '$a\,function(k,v,u){if(k!=="name"&&k!=="alpha2"&&isNaN(k)){return u}return v},2)' | node -p | sed -e '1s/^/countries = /' -e '$a\;' > countries.js)
```
Comment (jfly) [inline update-third-party-libs.sh]: What's `u` here? The docs on this replacer method say that the function only takes 2 arguments: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_native_JSON#The_replacer_parameter.

I think a ternary operator would be cleaner here:

```javascript
return (k!=="name"&&k!=="alpha2"&&isNaN(k)) ? u : v
```

Why do you need to check `isNaN(k)`?

**Comment by jfly** [comment]
Thanks for cleaning up the clean() method and tightening up our assert. One question about your script for you.

File: update-third-party-libs.sh (hunk 1 above)
Comment (Baiqiang) [inline update-third-party-libs.sh]: You're right, the method is passed with 2 arguments. So the 3rd argument will be undefined(u is short for undefined). I want it to be shorter.
And the isNaN(k) is to make sure the key is not an interger.

Comment (jfly) [inline update-third-party-libs.sh]: LOL, that's cute. This isn't a shortest code competition, though, lets go with just 2 arguments and undefined instead, please. You should get some extra characters from switching to a ternary operator anyways =).

When is the key an integer? Is this a convention I'm not familiar with?

Comment (Baiqiang) [inline update-third-party-libs.sh]: OK. I'll change it later. You see the countries is an array with object elements. Keys and values will be passed to the replacer recursive. For array, all keys are intergers.

**Comment by jfly** [comment]
LGTM. SHIP IT :rocket:

**Pull request merged**